dbus/Cargo.toml

```diff
@@ -28,15 +29,17 @@ winapi = { version = "0.3.0", features = ["winsock2"] }
 tempfile = "3"
 
 [features]
+default = ["macros"]
```

Comment:
Is there an important reason for this dependency? Can't dbus-macros (or what we rename it to) be its own standalone crate, like dbus-tokio, dbus-codegen, etc?

Reply:
Not really, mostly stylistic choice. I personally like the look of 1 dependency with a list of features. Fixed to follow your style